src/index.html

```diff
@@ -5,8 +5,340 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect"
+    href="https://fonts.googleapis.com"
+>
+    <link rel="preconnect"
+        href="https://fonts.gstatic.com"
+        crossorigin="-"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo-link" href="#">
+        <img class="header__picture"
+             src="images/logo.png"
+             alt="moyo's logo"
+        >
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link--is-active is-active" href="#">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+
+          <li class="nav__item" >
+            <a class="nav__link" data-qa="nav-hover"
+               href="#">Laptops & computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards-container">
+      <article class="card" data-qa="card">
```

Comment:
I don't think article is correct tag here, I think I would just go with `div` or `section`